apps/arm/ClawController.h

```diff
@@ -40,11 +40,14 @@ class ClawController : public ActuatorController {
 
     float m_calibrationTimeout_Seconds;
 
+<<<<<<< HEAD
+=======
     float convertShaftPositionDegreesToGapCm(float shaftPosition_Degrees);
     float convertShaftVelocityDegreesToGapVelocityCm(float shaftPosition_DegreesPerSec);
     float convertGapCmToShaftPositionDegrees(float gap_cm);
     float convertGapVelocityCmToShaftVelocityDegrees(float gap_cmPerSec);
```

Comment:
function names too long

Reply:
While this particular name is not very good, longer function names make code intent clearer and easier to read

Reply:
these names can be made shorter @wmmc88. Velocity -> Vel. Degrees -> Deg. Position -> Pos.

Reply:
In ROS, we follow REP 103 for standardized units and direction conventions. We have no equivalent for our fw repo so I believe @jetkov has set the precedent for ending function names with the unit.

Reply:
I rebased with alex's arm code at some point early on (I think), so thats why I have all this added. Not actually my code. I'll look up what REP 103 is tho.

Reply:
Sorry this is just one of my past co-op conventions. Yes they're long.